### DISCUSSION TITLE
Fill form field according to selection on another "option picker"

### DISCUSSION THREAD

**Discussion by jaarroyl** [Budibase/budibase#8411] [Help] [answered]
Hi guys!

Im new on budibase and I couldnt find anything about it.
I want to complete "Centro" field ones "Numero Sala" is filled by picker selection. ie >> 916
For doing that, I imagine that I could do some "vlookup" to a database (in this example I called test_Sala that has "id_sala" as Primary Key) using the "Numero Sala" filled value (916) to look in "test_Sala" and bring "Centro" value.

Can I do that?

![image](https://user-images.githubusercontent.com/41343847/198080139-f4a7f087-e1ca-40b3-bbb8-ad8a25943888.png)

![image](https://user-images.githubusercontent.com/41343847/198080293-00ce750c-ef60-4018-9e76-f5100d8d3805.png)

Cheers

**Comment by melohagan** [comment, marked as answer]
Hey @jaarroyl 

This tutorial shows how to retrieve relational data on change of a relationship picker: https://docs.budibase.com/docs/filter-by-relationships

Does that help get you started?

There is also the **On Change** event which could be useful: https://docs.budibase.com/docs/actions#on-change
